README.md

```diff
@@ -31,8 +30,10 @@ Fetches a given stemcell, placing the following files in the destination:
 
 * `version`: The version number of the stemcell.
 * `url`: A URL that can be used to download the stemcell tarball.
+* `sha1`: The SHA1 of the stemcell
 * `stemcell.tgz`: The stemcell tarball, if the `tarball` param is `true`.
 
 #### Parameters
 
 * `tarball`: *Optional.* Default `true`. Fetch the stemcell tarball.
+* `preserveFileName`: *Optional.* Default `false`. Keep the original filename of the stemcell.
```

Comment:
Params should be `snake_case`d.

Also: what kinds of filenames does this result in? `<stemcell>-<version>.tar.gz` would be nice.